.env.test

```diff
@@ -0,0 +1,14 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=999
+API_KEY=123
+TOKEN=456
+BOARD_ID = 1
+DEFAULT_TO_DO_NAME = 'To Do'
+DEFAULT_DOING_NAME = 'Doing'
+DEFAULT_DONE_NAME = 'Done'
```

Comment:
Quick pointer that will only really matter once you reach module 5:

The most correct syntax for the .env file is e.g. `DEFAULT_DOING_NAME=Doing`

No extra spaces, no quotation marks. This is not a Python file.

python-dotenv can load this fine, but Docker (which we use in Module 5) will not interpret this how you are expecting

Reply:
Resolved in module_4 https://github.com/ianm1968/ToDoTake2/commit/10760d33ef90f88f52dcb05aac6db3d48578f041